public/r2/v1/swagger/swagger.json

```diff
@@ -184,42 +184,42 @@
                 }
             }
         },
-        "/namespaces/{namespaceID}/ruleset/bulk": {
+        "/namespaces/{namespaceID}/ruleset/update": {
             "post": {
                 "tags":[
                     "namespaces"
                 ],
-                "summary": "Bulk update endpoint for ruleset changes",
-                "operationId": "bulkUpdateRuleSet",
+                "summary": "Update endpoint for a ruleset, allowing for multiple multiple atomic changes to a ruleset at once.",
```

Comment:
Delete second 'multiple'
Also super nit: for a public API mentioning multiple atomic changes could be confusing. Perhaps allowing for atomic changes for an entire RuleSet